website/docs/r/project_invitation.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: project_invitation"
+sidebar_current: "docs-mongodbatlas-resource-project-invitation"
+description: |-
+    Provides a Atlas Project Invitation resource.
+---
+
+# mongodbatlas_project_invitation
+
+`mongodbatlas_project_invitation` provides a Project Invitation resource. This represents an invitation for an Atlas User within an Atlas Project.
+
+Each invitation for an Atlas user has a set of roles that provide access to a project in an organization.
+
+The roles that can be utilised can be found in the [MongoDB Documentation](https://docs.atlas.mongodb.com/reference/user-roles/#project-roles), which map to:
+
+* GROUP_OWNER
+* GROUP_CLUSTER_MANAGER
+* GROUP_READ_ONLY
+* GROUP_DATA_ACCESS_ADMIN
+* GROUP_DATA_ACCESS_READ_WRITE
+* GROUP_DATA_ACCESS_READ_ONLY
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
+
+~> **IMPORTANT:** All arguments including the password will be stored in the raw state as plain-text. [Read more about sensitive data in state.](https://www.terraform.io/docs/state/sensitive-data.html)
+
+## Example Usages
+
+```hcl
+resource "mongodbatlas_project_invitation" "test" {
+  username    = "test-acc-username"
+  project_id  = "<PROJECT-ID>"
+  roles       = [ "GROUP_DATA_ACCESS_READ_WRITE" ]
+}
+```
+
+```hcl
+resource "mongodbatlas_project_invitation" "test" {
+  username    = "test-acc-username"
+  project_id  = "<PROJECT-ID>"
+  roles       = [ "GROUP_READ_ONLY", "GROUP_DATA_ACCESS_READ_ONLY" ]
+}
+```
+
+## Argument Reference
+
+* `project_id` - (Required) The unique ID for the project to create the database user.
```

Comment:
Same as above I'd do a bit more copy paste of the verbiage from our docs.  https://docs.atlas.mongodb.com/reference/api/project-create-one-invitation/

Reply:
I have put this in and will test in the next few days.